packages/e2e-tests/prebundle-svelte-deps/vite.config.js

```diff
@@ -0,0 +1,22 @@
+import { defineConfig } from 'vite';
+import { svelte } from '@sveltejs/vite-plugin-svelte';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+	plugins: [svelte()],
+	optimizeDeps: {
+		// TODO this must be excluded because nested has an scss dep that prebundle can't handle!
+		// figure out how to exclude it automatically or at least tell the user about it in a more friendly way
+		exclude: ['e2e-test-dep-svelte-nested'],
```

Comment:
Looks like a bug in Vite. Currently it externalizes `.scss`/`.css` etc files when prebundling, but looks like it's failing when it's a package that exports a `.scss` file. Replacing this as `e2e-test-dep-scss-only` should have the same effect